tapalcatl_server/server.go

```diff
@@ -333,15 +331,11 @@ func main() {
 	// metrics writer configuration
 	var mw metricsWriter
 	if metricsStatsdAddr != "" {
-		if metricsInterval <= 0 {
-			logger.Fatalf("ERROR: metrics interval should be greater than 0")
-		}
 		udpAddr, err := net.ResolveUDPAddr("udp4", metricsStatsdAddr)
 		if err != nil {
 			logger.Fatalf("ERROR: Invalid metricsstatsdaddr %s: %s", metricsStatsdAddr, err)
 		}
-		metricsIntervalDuration := time.Second * time.Duration(metricsInterval)
-		mw = NewStatsdMetricsWriter(udpAddr, metricsStatsdPrefix, metricsIntervalDuration)
+		mw = NewStatsdMetricsWriter(udpAddr, metricsStatsdPrefix, logger)
```

Comment:
Would a `Close()` function make sense here to close the channel feeding the goroutine? I _think_ the goroutine will be garbage collected along with the channel if the main thread exits... but I'm not entirely sure.